cluster-autoscaler/cloudprovider/azure/azure_scale_set.go

```diff
@@ -35,6 +35,18 @@ import (
 	"github.com/Azure/go-autorest/autorest/azure"
 )
 
+// PowerStates reflect the operational state of a VM
+// From https://learn.microsoft.com/en-us/java/api/com.microsoft.azure.management.compute.powerstate?view=azure-java-stable
+const (
```

Comment:
Could we move this to `azure_util.go`? 
Those are where the potentially sharable constants are located.

Let's also keep the naming consistent and descriptive about power state being from VM, something like `vmPowerStateStarting`.

Reply:
Sure thing, moved the constants and related helper functions to `azure_util.go`. I also adjusted the naming of the helper functions to better reference that it's a VM power state.